recipes/lsst-documenteer-pipelines/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "documenteer" %}  {# PyPI name #}
+{% set version = "0.5.6" %}
+
+package:
+  name: "lsst-documenteer-pipelines"
```

Comment:
This package could be a second output on the lsst-documenteer-recipe. That might help you pin against specific builds which could be useful for maintenance.

Reply:
Could you point me to docs or an example of this?

Reply:
Right so see the multiple outputs thing above. The pinnings you might want are easiest to achieve there.